tests/test_storage.py

```diff
@@ -332,6 +350,9 @@ def test_get_orders_live(storage_live):
     descending_dates = sorted(dates)[::-1]
     assert descending_dates == dates
 
+    orders_tags = storage_live.get_orders(tags=["Test"])
+    assert len(orders_tags) >= 0
```

Comment:
i was gonna ask why it is >= 0? do we expect 0 orders to be returned in the test

Reply:
Ah missed this. 
+1

Reply:
Hi guys, you are right. It should be more than 1. I will change this. 

What happens is:
Live tests should not be included, and we are slowly deprecating all of them. 

The results for the live test depend on our test account. I added only 1 order with a test tag.
Therefore, I added >= 0 in case the order tag is changed by accident to not cause test failure. 

My test currently is more tag parameter still being accepted by the endpoint, but you are right it should be that test filter that only orders with the `test` tag.